helm-go-lib/main.go

```diff
@@ -277,3 +275,59 @@ func (f *CustomConfigFlags) ToRESTConfig() (*rest.Config, error) {
 
 	return c, nil
 }
+
+//export pullChart
+func pullChart(chart string, chartVersion string, destDir string) *C.char {
+	var kubeConfig *genericclioptions.ConfigFlags
+	kubeConfig = genericclioptions.NewConfigFlags(false)
+
+	actionConfig := new(action.Configuration)
+	if err := actionConfig.Init(kubeConfig, "namespace", os.Getenv("HELM_DRIVER"), log.Printf); err != nil {
+		return nil
+	}
+
+	// Create a new client configuration
+	client := action.NewPull()
+	withConfig := action.WithConfig(actionConfig)
+	withConfig(client)
+
+	settings := cli.New()
+	client.Settings = settings
+
+	// TODO do we need this
+	// client.ChartPathOptions.RepoURL = "https://charts.helm.sh/stable"
```

Comment:
Do you still need that?

Reply:
I was worried about setting up the repo like with `helm repo add` ... But I am going to leave that out of this PR for now unless someone tells me otherwise.